projects/js-packages/components/components/a8c-svg-title/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * A8cSvgTitle component definition.
+ *
+ * @param {object} props - Component properties.
+ * @param {string} props.title - Title for SVG.
+ * @param {number} props.height - height for SVG.
+ * @param {number} props.className - className for the a wrapper, default: `jp-a8c-svg-title`.
+ *
+ * @returns {React.Component} A8cSvgTitle component.
+ */
+export default function A8cSvgTitle( {
+	title = 'An Automattic Airline',
```

Comment:
Could you make this translatable with `@wordpress/i18n` and a Jetpack textdomain?

Reply:
addressed in 05bed62